docs/merchant/overview.md

```diff
@@ -1,5 +1,19 @@
 ---
 id: overview
-title: Merchant Solutions
+title: Overview
 sidebar_label: Merchant Solutions
 ---
+
+We are developing a reference merchant project for developers. The project will demonstrate how your service can accept ≋Libra payments Libra online, in-app, and in-person.
+
+In this section, you will find the upcoming Libra Reference Merchant project, relevant technical and conceptual documentation, and technical specifications.
+
+Merchant services providers who would like to enable Libra payments on their platforms will need:
+  * To be a regulated Virtual Asset Service Provider (VASP). Review the requirements [here](https://libra.org/en-US/white-paper/#compliance-and-the-prevention-of-illicit-activity)
```

Comment:
Remove the `en-US` from all libra.org links.

Reply:
Yup. Sorry about that. Changing it in this PR
https://github.com/dfuzr/libra-website/pull/92/files